webpack.config.js

```diff
@@ -11,15 +11,16 @@ module.exports = {
     extensions: ['.js', '.jsx'],
   },
   devtool: 'eval-source-map',
-  module: {
-    rules: [
+  module : {
+    rules : [
       {
-        test: /\.jsx$/,
-        exclude: /node_modules/,
-        use: {
-          loader: 'babel-loader',
-        },
-      },
-    ],
-  },
+        test : /\.jsx?/,
+        exclude : /node_modules/,
+        loader : 'babel-loader',      
+        query: {
+          presets: ['react', 'es2015']

```

Comment:
I believe that the babel-preset-env is the updated version of babel-preset-es2015